packages/design/src/colors.css

```diff
@@ -46,10 +46,6 @@
   --color-surface--hover: var(--color-yellow--lightest);
   --color-surface--background: var(--color-grey--lightest);
   --color-surface--reverse: var(--color-greyBlue--dark);
-  --color-surface--success: var(--color-green--lighter);
```

Comment:
These surface-status values are now wrapped up in the status colors, as they should be